spec/system/story_submittal_and_retrieval_spec.rb

```diff
@@ -7,21 +7,25 @@
 require 'sidekiq/api'
 
 RSpec.describe 'story submittal and retrieval', sidekiq: true do
+  before do
+    # TODO: When form saving is fully functional consider enabling it here
```

Comment:
Why is this here? It's a feature toggle which is off by default.

Reply:
the tests for for asseted_views_spec were setting env variables, so depending on what was ran before here and while the form rembemering isn't working this is needed for consistency.